File: term2/week1-rs/exercise7.rs
use std::io;

fn main() {

    let mut width = String::new();
    println!("Enter the width: ");
    io::stdin()
        .read_line(&mut width)
        .expect("Failed to read, get good");
    let width = width.trim().parse::<i32>().unwrap();

    println!();

    for i in (0..(width+1)).rev() {
        for x in (1..(i+1)).rev() {
            print!("{}", x)
        }
        println!();
    }

}

File: term2/week1/week1-c/exercise7.c
#include <stdio.h>

int main() 
{
    int width;

    printf("Enter the width: ");
    scanf("%i", &width);

    for (int i = width-1; i != -1; i--)
    {
        for (int x = i+1; x != 0; x--) 
        {
            printf("%i ", x);
        }
        printf("\n");
    }
    return 0;
}
//  for i in (0..(width+1)).rev() {
//         for x in (1..(i+1)).rev() {
//             print!("{}", x)
//         }
//         println!();
//     }